data/templates/template.typst

```diff
@@ -35,55 +35,50 @@
     paper: paper,
     margin: margin,
     numbering: "1",
-  )
+    columns: cols,
+    )
+
   set par(justify: true)
   set text(lang: lang,
            region: region,
            font: font,
            size: fontsize)
   set heading(numbering: sectionnumbering)
 
-  if title != none {
-    align(center)[#block(inset: 2em)[
-      #text(weight: "bold", size: 1.5em)[#title]
-      #(if subtitle != none {
+  place(top + center, float: true, scope: "parent", clearance: 4mm)[
+    #if title != none {
+      text(weight: "bold", size: 1.5em)[#title]
```

Comment:
Can you say something about the rationale for this change in the way the title is formatted?

Reply:
sure: in versions <0.12 column placement was somewhat assumed to be global on layout; with typst >=0.12 it is explicitly controlled by page settings, with an explicit "scope", so the previous code placed only the main text in two columns, while the footnotes were still single column. Changing the page layout to double column had the side effect to set the title page also in double column (it was moved to the first, left column in my test). This new code is the current standard way to set a block of text single spaced, outside the scope of the double column page layout.